From: Maximiliano Sandoval <msandova@gnome.org>
Date: Thu, 20 Mar 2025 18:44:25 +0100
Subject: widget: Explicitly annotate sensitive's getter

This is a continuation of 91fe0ee63a66a8244b0fe7f68aa06dd25fb127a2.

Since
https://gitlab.gnome.org/GNOME/gobject-introspection/-/merge_requests/473
g-i will pick up gtk_widget_is_visible as the getter for this property.

See https://gitlab.gnome.org/GNOME/gjs/-/issues/681.

(cherry picked from commit a11310a40b7444b56823da6bbb44b759f2b15cc8)

Origin: upstream gtk-3-24 after 3.24.49
---
 gtk/gtkwidget.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/gtk/gtkwidget.c b/gtk/gtkwidget.c
index b6d125e..eb16d2e 100644
--- a/gtk/gtkwidget.c
+++ b/gtk/gtkwidget.c
@@ -1158,6 +1158,9 @@ gtk_widget_class_init (GtkWidgetClass *klass)
                             FALSE,
                             GTK_PARAM_READWRITE|G_PARAM_EXPLICIT_NOTIFY);
 
+  /**
+   * GtkWidget:visible: (getter get_sensitive)
+   */
   widget_props[PROP_SENSITIVE] =
       g_param_spec_boolean ("sensitive",
                             P_("Sensitive"),
